### ISSUE TITLE
The "more options" button is not keyboard accessible

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1392]
## Bug Report

**Problematic behavior**
Similarly, the "more options" button is not visible as it is hidden from assistive technologies. The user cannot access it using the keyboard.

**Expected behavior/code**
A clear and concise description of what you expected to happen (or code).

** Possible solution **

1. Remove the `aria-hidden="true"` attribute
2. Add an accessible name to the button `aria-label="More options"`

<img width="286" height="71" alt="Image" src="https://github.com/user-attachments/assets/26e7d802-24d8-44f0-b350-b37d4499a38f" />
